connection/cluster/ClusterDatabaseManager.java

```diff
@@ -59,21 +59,21 @@ public ClusterDatabaseManager(ClusterClient client) {
 
     @Override
     public boolean contains(String name) {
-        return failsafeTask(name, (stub, coreDbMgr) -> coreDbMgr.contains(name));
+        return failsafeTask(name, (stub, dbMgr) -> dbMgr.contains(name));
```

Comment:
rename `coreDbMgr` to just `dbMgr` here and in several other places, since it is not a Core-specific instance